### PR TITLE
chore: use externally published version of cdk-assets

### PR DESCRIPTION
**Pull request by TheRealAmazonKendra**
This PR only includes the removal of cdk-assets. In order to keep PRs small and clean, this does not include the documentation updates for how to find and contribute to cdk-assets for cloud-assembly-schema. That will be a separate PR.

### Checklist
- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by TheRealAmazonKendra** [comment]
Adding do-not-merge label while I run this through the test pipeline.

**Comment by TheRealAmazonKendra** [comment]
Removed the label as it passed the pipeline.

File: .github/workflows/request-cli-integ-test.yml [hunk 1]
```diff
@@ -27,13 +27,6 @@ jobs:
               - packages/aws-cdk/bin/**
               - packages/aws-cdk/lib/**
               - packages/aws-cdk/test/**
-              - packages/cdk-assets/bin/**
-              - packages/cdk-assets/lib/**
-              - packages/cdk-assets/test/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/lib/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/schema/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/scripts/**
-              - packages/aws-cdk-lib/cloud-assembly-schema/test/**
```
Comment (comcalvi) [inline .github/workflows/request-cli-integ-test.yml:36]: Cloud assembly schema is removed from here (and a few other places), but this PR doesn't delete that package and the title doesn't suggest that we're using the externally published version of cloud-assembly-schema as well. Can we update the PR title to reflect that?

Comment (TheRealAmazonKendra) [reply .github/workflows/request-cli-integ-test.yml:36]: Cloud assembly schema was removed in a previous PR.

**Review by comcalvi — APPROVED** [review]

**Pull request merged**